tests/end_to_end_tests/requirements.txt

```diff
@@ -1 +1,2 @@
 -r ../../tt_metal/python_env/requirements-dev.txt
+wheel
```

Comment:
Why is this necessary? It seems we are not installing the wheel package but how does it work now?

Reply:
Haha, why is this necessary?

What did you see in your testing that forced this? The wheel should come from `eager-package`, which doesn't seem to be run in a docker container still?

Reply:
I think these are leftovers from my testing - works now without it.